netbox/utilities/jinja2.py

```diff
@@ -48,10 +48,10 @@ def cache_templates(self, templates):
 # Utility functions
 #
 
-def render_jinja2(template_code, context):
+def render_jinja2(template_code, context, environment_params={}):
```

Comment:
`render_jinja2()` needs to be added to `__all__`. Not sure why it's not already there,